README.md

```diff
@@ -27,8 +27,8 @@ If you already have a Kentico Cloud account and you want to connect the sample a
 
 To preview content in the sample application, follow these steps:
 
-1. In Kentico Cloud, select your project.
-2. Navigate to the Development section.
+1. In Kentico Cloud, choose Project settings from the app menu.
+2. Under Development, choose API keys.
 3. Copy your Project ID and Preview API key.
```

Comment:
in KC it is a Delivery Preview API - do we want to change it?

Reply:
No, let's keep it as-is. It think it's straightforward because there's a single API to preview content and a single preview API key for that.